scripts/eval/yamls/tasks_v0.1.yaml

```diff
@@ -0,0 +1,266 @@
+icl_tasks:
+-
+  label: jeopardy
+  dataset_uri: eval/local_data/world_knowledge/jeopardy_all.jsonl # ADD YOUR OWN DATASET URI
+  num_fewshot: [10]
```

Comment:
Odd to have num_fewshot in both `tasks_v0.1.yaml` as well as `eval_gauntlet_v0.1.yaml`. Would it be possible to change that here? Might be out of scope.

Reply:
yeah I think its out of scope but son I'd like to merge it all into one yaml anyway